src/tree/updater_gpu_hist.cu

```diff
@@ -11,13 +11,17 @@
 #include "param.h"
 #include "updater_gpu_common.cuh"
 
+#define USE_NCCL 1
```

Comment:
Feel free to skip USE_NCCL, but if you want to make my life easy, please include it at least for now in the gpu_hist algo :)